hydra-tx/src/Hydra/Tx/Recover.hs

```diff
@@ -0,0 +1,54 @@
+module Hydra.Tx.Recover where
+
+import Hydra.Prelude
+
+import Hydra.Cardano.Api
+import Hydra.Cardano.Api.Network (networkIdToNetwork)
+import Hydra.Contract.Commit qualified as Commit
+import Hydra.Contract.Deposit qualified as Deposit
+import Hydra.Ledger.Cardano.Builder (
+  addInputs,
+  addOutputs,
+  emptyTxBody,
+  setValidityLowerBound,
+  unsafeBuildTransaction,
+ )
+import Hydra.Plutus.Extras (posixFromUTCTime)
+import Hydra.Tx (HeadId, headIdToCurrencySymbol)
+
+-- | Builds a recover transaction to recover locked funds from the v_deposit script.
+recoverTx ::
+  NetworkId ->
+  HeadId ->
+  -- | Deposit input
+  TxIn ->
+  -- | Already Deposited funds
+  [Commit.Commit] ->
+  -- | Recover deadline
+  UTCTime ->
+  -- | Lower bound slot number
+  SlotNo ->
+  Tx
+recoverTx networkId headId depositedTxIn depositted deadline lowerBoundSlot =
```

Comment:
Could: rename `depositedTxIn` to `depositTxIn`

This would separate the semantics more clearly between "the deposit" and "what was deposited".

Also it's not depositted, but deposited.